test/robot/replay/client.go

```diff
@@ -162,19 +164,21 @@ func doReplay(ctx context.Context, action string, in *Input, store *stash.Client
 		tracefile.System(),
 	}
 	cmd := shell.Command(gapit.System(), params...)
-	output, callErr := cmd.Call(ctx)
-	if err := worker.NeedsRetry(output, "Failed to connect to the GAPIS server"); err != nil {
-		return nil, err
-	}
-
+	outBuf := &bytes.Buffer{}
+	errBuf := &bytes.Buffer{}
 	outputObj := &Output{}
-	if callErr != nil {
-		if err := worker.NeedsRetry(callErr.Error()); err != nil {
+	if err := cmd.Capture(outBuf, errBuf).Run(ctx); err != nil {
+		if err := worker.NeedsRetry(err.Error()); err != nil {
 			return nil, err
 		}
-		outputObj.CallError = callErr.Error()
+		outputObj.Err = err.Error()
 	}
-	output = fmt.Sprintf("%s\n\n%s", cmd, output)
+	outputObj.Err += strings.TrimSpace(errBuf.String())
```

Comment:
Seems like this will cause the strings to join back-to-back with no newlines?

Maybe use a `strings.Join(errs, "\n")`, or add some sort of separator here?